docs/architecture/solana/migration/blockchain-api.mdx

```diff
@@ -10,10 +10,9 @@ slug: /solana/migration/blockchain-api
 
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-The existing [Helium API](/blockchain/api) will serve only information up to the chain halt at the
-time of migration. Knowing that some users will continue to rely on historical data, The Helium
-Foundation has formed an agreement with the API provider, BlockJoy, to continue hosting this API
-following the migration.
+The existing Helium API will serve only information up to the chain halt at the time of migration.
+Knowing that some users will continue to rely on historical data, The Helium Foundation has formed
+an agreement with the API provider, BlockJoy, to continue hosting this API following the migration.
```

Comment:
Needs content updates.

Reply:
minimal update, these migration paged are not intended to last forever as is